lang/ja.json

```
@@ -397,84 +397,84 @@
       }
     },
     "badges": {
-      "null": "No Badge",
+      "null": "バッジなし",
       "locked": "???",
       "location": "Location:&nbsp;",
```

```suggestion
      "location": "場所：",
```
Should it be like this?

---

yes
thanks